common/app/common/configuration.scala

```diff
@@ -195,6 +195,7 @@ class GuardianConfiguration(val application: String, val webappConfDirectory: St
 
   object images {
     lazy val path = configuration.getMandatoryStringProperty("images.path")
+    val fallbackLogo = Static("images/fallback-logo.png").path
```

Comment:
Just worth noting the size of fallback-logo.png is currently optimised for facebook.